doc/api/cli.md

```diff
@@ -121,6 +121,16 @@ added: v6.0.0
 
 Print stack traces for process warnings (including deprecations).
 
+### `--redirect-warnings=file`
+<!-- YAML
+added: REPLACEME
+-->
+
+Write process warnings to the given file instead of printing to stderr. The
```

Comment:
A filepath only (absolute or relative). We could theoretically allow for an fd number but not sure if making it more complicated is worth it.... not out of the question tho.